src/pages/Main/components/PostList/AllOfPostList.tsx

```diff
@@ -0,0 +1,10 @@
+import PostList from '@/pages/Main/components/PostList/PostList';
+import useFetchAllOfPosts from '@/pages/Main/components/hooks/useFetchAllOfPosts';
+
+const AllOfPostList = () => {
+  const response = useFetchAllOfPosts({ limit: 9 });
```

Comment:
이부분이 마지막에 읽으시면 좋을 리뷰입니다 !

이전에 뭔가 우수수 적어놓은걸 정리하는 것 같아서요 ! 

```suggestion
  const {data,  fetchNextPage, hasNextPage, isFetchingNextPage } = useFetchAllOfPosts({ limit: 9 });
  
  return <PostList 
  data = {data} 
  fetchNextPage = {fetchNextPage} 
  hasNextPage = {hasNextPage}   
  isFetchingNextPage = {isFetchingNextPage}
  />;
```

이렇게 해서 사용하면 컴포넌트에서 커스텀훅을 호출하지 않아도 되겠네요 ! 
반드시 꼭 ! 마지막에 읽어주세요.. 이전 리뷰랑은 내용이 살짝 달라서요 ... ㅎ

각 타입은 [여기](https://tanstack.com/query/latest/docs/framework/react/reference/useInfiniteQuery?from=reactQueryV3)에 명시되어있습니다 :)

Reply:
위에서 말씀드린데로

```ts

const AllOfPostList = () => {
  const response = useFetchAllOfPosts({ limit: 9 });

  return <PostList {...response} />;
};
```

이런식으로 수정해봤는데, 괜찮을까용?!

Reply:
이거 한번 저랑 따로 코드보면서 얘기해보시죠 !